Library/ViewModels/DiscoveryPageViewModel.swift

```diff
@@ -331,7 +331,7 @@ public final class DiscoveryPageViewModel: DiscoveryPageViewModelType, Discovery
     )
 
     self.configureEditorialTableViewHeader = paramsChanged
-      .filter { $0.tagId == .goRewardless }
```

Comment:
Actually I noticed while testing that the light's on projects view has the "These projects could use your support" header -- it looked a bit odd, I'm not sure if we want to keep it?

Reply:
Ah, yes forgot to remove that, thanks.